apps/salesforce-commerce-cloud/package.json

```diff
@@ -28,7 +28,7 @@
     "build": "tsc && vite build",
     "test": "vitest",
     "deploy": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEFINITIONS_ORG_ID} --definition-id 1O8njPYTGO44VueSJNBrdZ --token ${CONTENTFUL_CMA_TOKEN}",
-    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${TEST_ORG_ID} --definition-id 66J3nYAMa13A1O7E8MDDMu --token ${CONTENTFUL_CMA_TOKEN}"
+    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEV_TESTING_ORG_ID} --definition-id 66J3nYAMa13A1O7E8MDDMu --token ${TEST_CMA_TOKEN}"
```

Comment:
These now reflect the variables we have setup in circle ci for other apps staging deploys